pkg/actuator/info/endpoint.go

```diff
@@ -19,6 +19,8 @@ package info
 import (
 	"context"
 
+	"mosn.io/layotto/pkg/filter/stream/common/http"
+
 	"mosn.io/pkg/log"
 
 	"mosn.io/layotto/pkg/actuator"
```

Comment:
format this file

Reply:
There are also some other files that need to be formatted

Reply:
what do you mean，gofmt already executed, there is any special code style？

Reply:
Just delete the empty lines and rerun go fmt

Reply:
i think the empty lines in import section is necessary，and it's the rule for goImports in the CI process.

Reply:
the imported packages under the same mod do not need empty lines